packages/fxa-payments-server/src/lib/stripe.ts

```diff
@@ -137,6 +137,9 @@ export async function handleSubscriptionPayment({
     await stripe.createPaymentMethod({
       type: 'card',
       card: card as StripeCardElement,
+      billing_details: {
+        name,
```

Comment:
Can we add the email address here too? Stripe will also use that....

Reply:
Sounds good. Latest changes added the email address as well. Some test users are [here](https://dashboard.stripe.com/test/customers/cus_NsLsnWAVa6SHV7) and [here](https://dashboard.stripe.com/test/customers/cus_NsLtOlFXJkaLS4). Seems to work as expected.